app/assets/stylesheets/booklists.css.scss

```diff
@@ -1,3 +1,7 @@
 .booklist-select {
   margin-bottom: 10px;
 }
+
+.booklist-info-red {
+  color: #d9534f;
```

Comment:
Color literals like `#d9534f` should only be used in variable declarations; they should be referred to via variable everywhere else.